instance/migrations/0053_instancemanagerpermission_data.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-06-07 06:13
+from __future__ import unicode_literals
+
+from django.db import migrations
+from instance.permissions.models import InstanceManagerPermission
+
+def create_instance_manager_permission(apps, schema_editor):
+	"""Create the singleton InstanceManagerPermission"""
+	InstanceManagerPermission.objects.get_or_create(codename=InstanceManagerPermission._meta.verbose_name)
```

Comment:
@smarnach I agree with @bradenmacdonald on this one.  The only hazy part of the use of this permission is that we're using it to restrict access to all of /api, not just /api/v1/instance.

Other than that, it really is just a restriction on access to the "instance" models, and since InstanceReference is shared by all of them, it's not a stretch to use that instead of a proxy model.  Also, when we want to add new "owned instance" permissions, it will be straightforward.

Also means a significantly smaller diff.  Please see [de57bdf](https://github.com/open-craft/opencraft/pull/104/commits/de57bdfed33b71bb0e4eb1315e6e0656b7d1c2f5) and let me know what you think?